src/regedit/register.jl

```diff
@@ -1,42 +1,8 @@
-"""
-Return a `GitRepo` object for an up-to-date copy of `registry`.
-"""
-function get_registry(registry::AbstractString; gitconfig::Dict=Dict())
```

Comment:
This is used here: https://github.com/JuliaRegistries/Registrator.jl/blob/03f4fa472ac6016e168271994110f9b1b53619ab/src/regedit/register.jl#L470

and here: https://github.com/JuliaRegistries/Registrator.jl/blob/03f4fa472ac6016e168271994110f9b1b53619ab/src/regedit/register.jl#L475

Reply:
Oh, I see that it is in types.jl. Nevermind.